### PR TITLE
Support cfq scheduler

### PR DESCRIPTION
**Pull request by dzhengfy**
Although cfq is out of date, we still need make existing cases can
work with cfq to avoid regression.

```
Before fix:
# avocado run --vt-type libvirt --vt-machine-type q35 type_specific.io-github-autotest-libvirt.virsh.blkiotune.positive.set_blkio_parameter.running_guest.change_weight.options.live.minimum_boundary
 (1/1) type_specific.io-github-autotest-libvirt.virsh.blkiotune.positive.set_blkio_parameter.running_guest.change_weight.options.live.minimum_boundary: ERROR: [Errno 2] No such file or directory: '/sys/fs/cgroup/blkio/machine.slice/machine-qemu\\x2d1\\x2davocado\\x2dvt\\x2dvm1.scope/blkio.bfq.weight' (65.36 s)
RESULTS    : PASS 0 | ERROR 1 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 69.20 s
```

### CODE REVIEW

**Comment by dzhengfy** [comment]
Depends on https://github.com/avocado-framework/avocado-vt/pull/3010

**Review by lento-sun — APPROVED** [review]
LGTM

File: virttest/libvirt_cgroup.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
 from virttest import virsh
 from virttest.staging import utils_cgroup
+from virttest import utils_disk
```
Comment (lento-sun) [inline virttest/libvirt_cgroup.py:14]: just one thing the imported modules should be arranged alphabetically according to the clean code instruction 
so the 'utils_disk' could be imported before 'virsh'

Comment (dzhengfy) [reply virttest/libvirt_cgroup.py]: updated.

File: virttest/libvirt_cgroup.py [hunk 2]
```diff
@@ -154,10 +156,12 @@ def __get_cg_file_path(cg_key, cg_path, cg_file_name):
             dev_init_dict = {"rbps": "max", "wbps": "max", "riops": "max",
                              "wiops": "max"}
             dev_list = []
+            if not self.is_bfq():
+                CGROUP_V1_BLKIO_FILE_MAPPING['weight'] = 'blkio.weight'
             for cg_key, cg_file_name in list(CGROUP_V1_BLKIO_FILE_MAPPING.items()):
                 cg_file_path = __get_cg_file_path(cg_key, cgroup_path, cg_file_name)
                 with open(cg_file_path, 'r') as cg_file:
-                    if cg_key in ["weight"]:
+                    if cg_key in ["weight", "weight_device"]:
```
Comment (lento-sun) [inline virttest/libvirt_cgroup.py:164]: the weight_device is like other device specific param such as 'device_read_iops_sec', so I guess it should be put in the "if cg_key in ["rbps", "wbps", "riops", "wiops"]:" branch.
And the device recorded as the key in the standardized dict in the format of dev's major:minor number. 
for example, our current test info as follow:
`2021-04-22 03:55:25,997 control_cgroup   L0229 DEBUG| blkiotune: The input info is: {'252:0': {'rbps': '345600000', 'wbps': 'max', 'riops': 'max', 'wiops': 'max'}}The ouput info is: {'weight': '100', '252:0': {'rbps': '345600000', 'wbps': 'max', 'riops': 'max', 'wiops': 'max'}}The cgroup info is:{'weight': '100', '252:0': {'rbps': '345600000', 'wbps': 'max', 'riops': 'max', 'wiops': 'max'}}`
the 252:0 is the device such as the /dev/sda

**Comment by dzhengfy** [comment]
Test result, see https://github.com/autotest/tp-libvirt/pull/3432#issuecomment-828896556

**Comment by dzhengfy** [comment]
@lento-sun Could you help review again? I updated according to your suggestion.

File: virttest/libvirt_cgroup.py [hunk 3]
```diff
@@ -154,12 +155,15 @@ def __get_cg_file_path(cg_key, cg_path, cg_file_name):
             dev_init_dict = {"rbps": "max", "wbps": "max", "riops": "max",
                              "wiops": "max"}
             dev_list = []
+            if not self.is_bfq():
+                CGROUP_V1_BLKIO_FILE_MAPPING['weight'] = 'blkio.weight'
+                CGROUP_V1_BLKIO_FILE_MAPPING['weight_device'] = 'blkio.weight_device'
```
Comment (lento-sun) [inline virttest/libvirt_cgroup.py:160]: great, it's better to test weigt_device only in cfq env. this is a simple fix and won't effect cgroup1/2 bfq env.

**Comment by lento-sun** [comment]
> @lento-sun Could you help review again? I updated according to your suggestion.

LGTM

**Review by chunfuwen — APPROVED** [review]
lgtm

**Review by kamilvarga — COMMENTED** [review]
LGTM.

**Pull request merged**